### ISSUE TITLE
Rendering is not working

### ISSUE THREAD

**Issue by sebastian-gerdes** [rstudio/tensorflow.rstudio.com#25]
Dear RStudio team,

thanks a lot for providing the tutorials. This is a good starting point for me.

I copied the source file into a new quarto document in RStudio. When I run the chunks individually, everything is fine. However, when I hit the "render" button, the chunk in which the layers are set up seems to cause a problem - the process is terminated without reporting a specific cause (see screenshot). I have had the same problem with other tutorials as well. 

This is the code that seems to be causing the problem:
`
model <- keras_model_sequential()
model %>%
  layer_flatten(input_shape = c(28, 28)) %>%
  layer_dense(units = 128, activation = 'relu') %>%
  layer_dense(units = 10, activation = 'softmax')
`

When I run `knit('basic_image_classification.qmd')` from the console, a *.txt in markdown is compiled which I can then render interactively in RStudio after having changed the suffix to *.md.  

Any ideas what could be causing the problem?

Any help is greatly appreciated!

Best greetings,
Sebastian

PS: Just a suggestion - it could be helpful to give names to the code chunks - this would make talking about the code easier! 

<img width="1552" alt="Bildschirmfoto 2022-08-03 um 08 39 31" src="https://user-images.githubusercontent.com/61906380/182541978-f4fc2ae6-8921-4cac-b372-5eaf61bb1d0f.png">

**Comment by t-kalinowski** [comment]
Naming code chunks is a great idea. Contributions welcome :)

To render the documents at the console you'll have to use quarto, either from the terminal like this:
```bash
quarto render tutorials/keras/classification.qmd
```
or from the R console using the quarto package:
```R
library(quarto)
quarto_render("tutorials/keras/classification.qmd")
```

**Comment by sebastian-gerdes** [comment]
Dear Tomasz,

thanks for your answer! The problem was also due to an old version of tensorflow in the environment I was working in I just figured out.

I am kind of new to git - what is the best way to contribute / suggest changes to an individual file? Can I somehow "pull" it in my RStudio environment and send something like a "pull request" with the changes to you? I hope I am not "messing around" too much with the git terminology.

Best greetings,
Sebastian

**Comment by t-kalinowski** [comment]
No worries, the git terminology is certainly confusing at first, but becomes clearer with a little explanation. You might in enjoy this getting-started guide that shows you how to open your first PR to an R package: https://usethis.r-lib.org/articles/pr-functions.html
